react/features/base/premeeting/components/web/CopyMeetingUrl.js

```diff
@@ -36,6 +36,11 @@ type State = {
 
 const COPY_TIMEOUT = 2000;
 
+/**
+ * Set to true in order to automatically copy invitation link after creating a meeting.
+ */
+const ENABLE_AUTOMATIC_URL_COPY = false;
```

Comment:
Config goes in config.js, at least commented. Code should work with setting missing in conifg.js. And if it will be used to set it through iframe api or url, needs to be added to whitelist of configs that are allowed to be overwritten. Thanks

Reply:
Added to config file